backend/lib/azimutt_web/views/organization_view.ex

```diff
@@ -5,4 +5,134 @@ defmodule AzimuttWeb.OrganizationView do
     {:ok, relative_str} = datetime |> Timex.format("{relative}", :relative)
     relative_str
   end
+
+  def event_description(%{name: "editor_layout_created", created_by: user, project: project}) do
```

Comment:
Looks like the only difference is the `action`, why not only return it? (and take `author` and `destination` externally).
Also, transforming event name in `action` looks like business logic, so could be done inside a service rather than view is we want to reuse it somewhere, maybe `Tracking` or an other one...